libs/common/src/change-stream/test-helpers/generate-mock-audit-database-record.ts

```diff
@@ -48,21 +44,24 @@ export const generateParsedMockTfmUserAuditDatabaseRecord = (mockUserId: string
  * @returns mock data that an api GET request would return.
  * In particular, the ObjectId is converted to strings
  */
-// eslint-disable-next-line @typescript-eslint/no-unsafe-return
 export const generateParsedMockPortalUserAuditDatabaseRecord = (mockUserId: string | ObjectId) => ({
   ...JSON.parse(JSON.stringify(generatePortalUserAuditDatabaseRecord(mockUserId))),
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
   lastUpdatedAt: expect.any(String),
 });
 
+export const expectAnyPortalUserAuditDatabaseRecord = () => ({
+  lastUpdatedAt: expect.any(String),
+  lastUpdatedByPortalUserId: expect.anything(),
+  lastUpdatedByTfmUserId: null,
+  lastUpdatedByIsSystem: null,
+  noUserLoggedIn: null,
+});
+
 export const generateParsedMockAuditDatabaseRecord = (auditDetails: AuditDetails) => {
```

Comment:
I dont know why this is showing here (maybe you picked this commit) but worth making sure this is updated to latest